apps/kudosbot-server/src/routing/routes/uploadRoute.ts

```diff
@@ -12,6 +12,14 @@ export function initUploadRoute(
 ): void {
   console.log("[INIT] Initializing upload route");
 
+  const deserializeKudosData = (kudosData: string) => {
```

Comment:
all the logic in this file should go within a kudosRoutes.ts file in passport-server